multitenant_provider/pyproject.toml

```diff
@@ -5,13 +5,20 @@ description = ""
 authors = ["Jason Sherman <tools@usingtechnolo.gy>"]
 
 [tool.poetry.dependencies]
-aries-cloudagent = { version = "0.10.3" }
 bcrypt = "^4.0.1"
 mergedeep = "^1.3.4"
 python = "^3.9"
 python-dateutil = "^2.8.2"
 typing-extensions = "4.5.0"
 
+# Define ACA-Py as an optional/extra dependancy so it can be
+# explicitly installed with the plugin if desired.
+aries-cloudagent = { version = "0.10.3" }
```

Comment:
here too

Reply:
The `multitenant_provider` didn't like it when it was updated to a newer version of ACA-Py, the tests failed, so I reverted it back to the version that worked.
See:
- https://github.com/hyperledger/aries-acapy-plugins/actions/runs/7090862684/job/19298755060
- https://github.com/hyperledger/aries-acapy-plugins/actions/runs/7090862685/job/19298755070